common/src/main/java/org/schemarepo/config/Config.java

```diff
@@ -90,6 +105,9 @@
 
     DEFAULTS.setProperty(DEFAULT_SUBJECT_VALIDATORS,"");
 
+    // Validation defaults. The read strategy class is "Always Fail" unless configured.
+    DEFAULTS.setProperty(VALIDATION_READ_STRATEGY_CLASS, "org.schemarepo.validation.AlwaysFailValidationStrategy");
```

Comment:
This is purely my opinion, but to answer your two questions (I'm sure others probably have other opinions also)
1. No, I imagine the 'CanReadStrategy' will be either Avro or "something else"... it's really the core of the definition of "test if one schema can read another one"...
2. I personally don't feel like having a simplified "validation.type=avro" would be useful (especially if it were "mutual read for every schema"). I feel like each company should discuss and come to a policy on how they want to validate. For example with out company, we copy our avro data to HDFS and keep it for an indefinite amount of time, so "CanReadAll" is important to us. However "CanBeReadByAll" is simply too restrictive, as we control all of our readers and can upgrade them all, so simple "CanBeReadByLatest" is good enough for us to ensure we can have a mixed-mode environment during a rolling upgrade. To some I imagine this wouldn't be necessary at all. I just don't know what use scenarios others would have.

That being said, I'm totally fine with you coming up with some defaults as long as we could still achieve what we are looking for (CanReadAll, CanBeReadByLatest)

Thanks again for going through all this.
Thunder